contrib.php

```diff
@@ -15,31 +15,20 @@
 // multiuser permission check
 permission_or_die(PERM_WRITE);
 
-
 /**
  * Sort multi-dimensional arrays 
  * (thanks to phpdotnet)
  */
 function multidimsort($array_in, $column)
 {
-    $multiarray = array();
+    $multiarray = array_column($array_in, $column);
     $array_out  = array();
     $loopvalue  = 0;
 
-    // -1 as traversal of array starts from 0, count() starts from 1
-    $multicount = count($array_in) - 1;
-
-    for($i = 0; $i <= $multicount; $i++)
-    {
-        array_push($multiarray, $array_in[$i][$column]);
-    }
     asort($multiarray);
 
-    // reset internal pointer to beginning of array after above sort
-    reset($multiarray);
-
     // traverse new array of index values and add the corresponding element of the input array to the correct position in the output array
-    while (list ($key, $val) = each ($multiarray))
+    foreach ($multiarray as $key => $val)
     {
         $array_out[$loopvalue] = $array_in[$key];
```

Comment:
One last thing... pls remove the loopvalue var and just do

    $Ary[] = $appendedvalue

And I do realize I wrote much of this code ...

Reply:
@andig Jo ist angepasst, das hast du ganz schön verkackt 😉 Kleiner Scherz! 😉

So viel sollte ja jetzt nicht mehr von der Methode übrig sein was man optimieren kann 😉